tincr/io/design/tincr_checkpoints.tcl

```diff
@@ -389,7 +392,7 @@ proc ::tincr::write_macros { {filename macros.xml } } {
 ## Creates the "placement.rsc" file within a TINCR checkpoint targeting RapidSmith
 #
 # @param filename The name of the placement checkpoint file, "placement.rsc" is the default.
-proc ::tincr::write_placement_rs2 { {filename placement.rsc} }  {
+proc ::tincr::write_placement_rs2 { is_series7 {filename placement.rsc} }  {
```

Comment:
Rather than passing in `is_series7`, the `write_placement_rs2` and `write_routing_rs2` procs should compute whether or not the target device is series 7 by calling into a helper proc. IOW move the logic you're using to set `is_series7` into a proc and have both procs call into that.